doc/classes/ParticlesMaterial.xml

```diff
@@ -294,152 +297,217 @@
 	</methods>
 	<members>
```

Comment:
Something is wrong with getters and setters. It feels strange to have `set_param_texture` for `angle_curve` member... It may be a bug in the docs XML splitter.

Reply:
I don't think it's an XML bug. Every property category (velocity, angle, etc) has three param setters: `set_param`, set_param_texture`, and `set_param_randomness` to set the value, curve, and randomness. 

I do agree it's really verbose, and I am not sure why you would use them over the members.

Reply:
It seems normal indeed: this is the actual prototype `set_param("parameter_name", value)`.
It may be unclear that this method takes the parameter to be set. I wonder if we can do something about this @NathanLovato @reduz @akien-mga  ?

Reply:
It seems these were overwritten? Can't find the line anymore. If it's okay now, can you approve the changes?

Reply:
I think we should merge as-is. If the property setters change later, we can revise then.